components/announcement_bar/text_dismissable_bar.test.jsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import TextDismissableBar from 'components/announcement_bar/text_dismissable_bar.jsx';
+
+describe('components/TextDismissableBar', () => {
+    const baseProps = {
+        allowDismissal: true,
+        text: 'sample text',
+        siteURL: 'http://testurl.com',
```

Comment:
should there be a test where `siteURL` is not present? i.e. null/undefined? since this field is defined as `siteURL: PropTypes.string,` i.e. not required.

Reply:
fair point

Reply:
done